### ISSUE TITLE
Activity log not registering in 3DS

### ISSUE THREAD

**Issue by KiteAzureFlame** [ScarletStudy/DGS1-3DS-Release#105]
Hello! First, I would like to thank you all for this wonderful english translation! As a great fan of the Ace Attorney series, being able to play a new game made me so happy that is impossible to describe in words. Now talking about the issue, I noticed that The Great Ace Attorney after being english patched does not show in the 3DS activity log. I can't see my playtime or how many times I played the game, etc, since it does not show there in the rankings. I really like to see how much time I invested in my 3DS library so not being able to see it for this game is a big deal for me. Is there a way to fix that? Am I the only one with this problem? Thank you in advance!!

**Comment by wabberz** [comment]
Hm, I don't think we've done anything that would influence this. There's a flag in the Application Settings header that could disable it, we didn't touch it though. But to be perfectly honest, I haven't tested this and you're the first one to bring this up. Are you sure your playtime is not logged? The title should still be in Japanese.

**Comment by KiteAzureFlame** [comment]
Yes, I'm sure it's not being displayed. I use an CFW to be able to play japanese games and this is the only title that doesn't show there. I looked at the log imediately after playing it, but it did not showed up. Anyway, thank you for the quickly reply.

**Comment by ederenzi78** [comment]
Just to say that for my New 3DS XL it correctly shows up in the activity log.
System: N3DSXL EUR, latest official firmware as of today (3 February 2020)
CFW: Luma3DS 10.0.1. I've set the locale to JP country / JP language.
Patch version: 2.4.1
Using original cart
